src/RiakClient/Models/MapReduce/RiakMapReduceResultPhase.cs

```diff
@@ -28,10 +28,10 @@ namespace RiakClient.Models.MapReduce
     public class RiakMapReduceResultPhase
     {
         private readonly bool success;
-        private readonly uint phase;
+        private readonly long phase;
```

Comment:
PB interfaces uses `uint`s, best leave it there so we don't run into any crazy casting problems in the future. It's a consume only property/variable too, so shouldn't be a problem.